File: Pemrograman1/Modul4/Python/PRAK403-2310817310009-MuhammadFauzanAhsani.py
var = input().split()
val1, val2 = map(int, var)

for i in range(abs(val1 - val2) + 1):
    if (val1 < val2):
        if (i == abs(val1 - val2)):
            print(f"{val1 + i} {val2 - i}", end = "")
        else:
            print(f"{val1 + i} {val2 - i} - ", end = "")
    elif (val1 > val2):
        if (i == abs(val1 - val2)):
            print(f"{val1 - i} {val2 + i}", end = "")
        else:
            print(f"{val1 - i} {val2 + i} - ", end = "")

File: Pemrograman1/Modul4/C/PRAK403-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>
#include <math.h>

int main(){
    int val1, val2;

    scanf("%d %d", &val1, &val2);

    for (int i = 0; i < (abs(val2 - val1) + 1); i++){
        if (val1 < val2){
            if (i == (abs(val2 - val1))){
                printf("%d %d", val1 + i, val2 - i);
            }
            else {
                printf("%d %d - ", val1 + i, val2 - i);
            }
        }
        else if (val1 > val2){
            if (i == (abs(val2 - val1))){
                printf("%d %d", val1 - i, val2 + i);
            }
            else {
                printf("%d %d - ", val1 - i, val2 + i);
            }
        }
    }
    return 0;
}